system/jlib/jmalloc.cpp

```diff
@@ -1178,6 +1178,8 @@ class CSuperAllocator: public CInterface, implements IAllocator
             *   3. Put a 'free' tag on the end.
             *   4. Change the size recorded in the block.
             */
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
```

Comment:
PrevFreeEnt->Size += CurBlockSize;

Pretty sure it's a false positive and there's no way to get to that call without it being previously set.  But it gets set in a a previous if() block that uses part of the same set of conditionals.